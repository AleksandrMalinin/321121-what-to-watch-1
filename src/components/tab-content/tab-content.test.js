import React from 'react';
import renderer from 'react-test-renderer';
import TabContent from './tab-content.jsx';

it(`TabContent correctly renders`, () => {
  const tree = renderer
  .create(<TabContent/>)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
