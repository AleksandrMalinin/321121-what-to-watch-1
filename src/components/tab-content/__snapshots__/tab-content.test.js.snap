// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabContent correctly renders 1`] = `
<div
  className="movie-card__reviews movie-card__row"
/>
`;
